### PR TITLE
JSDocs for src/react directory

### PR DESCRIPTION
**Pull request by tbarlow12**

### CODE REVIEW

File: src/react/components/common/conditionalNavLink/conditionalNavLink.tsx [hunk 1]
```diff
@@ -1,6 +1,17 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
 
+/**
+ * Link able to be enabled/disabled
```
Comment (mydiemho) [inline src/react/components/common/conditionalNavLink/conditionalNavLink.tsx]: what is link able?

**Comment by mydiemho** [comment]
@tbarlow12  we should try to do smaller PR (not just complexity but also line changes).  everything seem to blend together after 10 files

File: src/react/components/common/filePicker/filePicker.tsx [hunk 2]
```diff
@@ -2,11 +2,20 @@ import React, { SyntheticEvent } from "react";
 import shortid from "shortid";
 import HtmlFileReader from "../../../../common/htmlFileReader";
 
+/**
+ * Properties for File Picker
+ * @member - Function to call on change of file selection
+ * @member - Function to call on file picking error
+ */
```
Comment (luisamiranda) [inline src/react/components/common/filePicker/filePicker.tsx]: names of functions - onChange and onError - missing here.

File: src/react/components/common/messageBox/messageBox.tsx [hunk 3]
```diff
@@ -1,8 +1,19 @@
 import React, { SyntheticEvent } from "react";
 import { Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
 
+/**
+ * Accepts any number of parameters and return a string
+ */
```
Comment (luisamiranda) [inline src/react/components/common/messageBox/messageBox.tsx]: "returns" a string

**Comment by luisamiranda** [comment]
Made a couple of comments, but both non-essential fixes, so I approved anyway.

File: src/react/components/common/conditionalNavLink/conditionalNavLink.tsx (hunk 1 above)
Comment (tbarlow12) [reply src/react/components/common/conditionalNavLink/conditionalNavLink.tsx]: You can enable or disable the Link

File: src/react/components/common/filePicker/filePicker.tsx (hunk 2 above)
Comment (tbarlow12) [reply src/react/components/common/filePicker/filePicker.tsx]: Good catch :)

**Pull request merged**